pkg/networkValidator/validator_test.go

```diff
@@ -41,6 +51,11 @@ func test_identifyNetworkError_network_error(validator *NetworkValidator, offlin
 	return validator.IdentifyNetworkError("network error")
 }
 
+func test_identifyNoInternet_network_error(validator *NetworkValidator, offlineMode string) error {
+	validator.SetOfflineMode(offlineMode)
+	return validator.IdentifyNetworkError("no such host")
```

Comment:
it's worth adding tests for the other errors that indicate a network error

Reply:
We have 2/3. I thought it's not the important to add the connection refused one